data/mods/Aftershock/recipes/frankenstein_recipes.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "type": "recipe",
+    "result": "inactive_frankenstein",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_CBMS",
+    "skill_used": "electronics",
+    "skills_required": [ "firstaid", 8 ],
+    "difficulty": 7,
+    "time": 50000,
+    "reversible": true,
+    "decomp_learn": 5,
+    "book_learn": [ [ "cyrus'_notes", 7 ] ],
+    "using": [ [ "soldering_standard", 20 ] ],
+    "tools": [ [ [ "afs_solderers_item", -1 ], [ "toolset", -1 ] ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 } ],
+    "components": [
+      [ [ "power_supply", 6 ], [ "UPS_off", 1 ] ],
```

Comment:
Should probably use one or two pieces of "afs_energy_storage_4" instead of the power supply.  After all making an npc is a considerably strong ability

We would need to add spawns for that part however, which would imply a bit of work.  I could help you with that if you agree.

Reply:
I completely agree.  There are several places I'm leaving old style items until some things get merged post 0.E.  I've basically got a list in my head of the order in which my open PRs need to get processed.